tests/wallet/test_wallet.py

```diff
@@ -762,95 +761,194 @@ async def test_clawback_resync(
         # Check merkle coins
         await time_out_assert(
-            20, wallet_node.wallet_state_manager.coin_store.count_small_unspent, 1, 1000, CoinType.CLAWBACK
+            20, wallet_node_1.wallet_state_manager.coin_store.count_small_unspent, 1, 1000, CoinType.CLAWBACK
         )
-        assert await wallet.get_confirmed_balance() == 3999999999500
+        await time_out_assert(
+            20, wallet_node_2.wallet_state_manager.coin_store.count_small_unspent, 1, 1000, CoinType.CLAWBACK
+        )
+        tx2 = await wallet_1.generate_signed_transaction(
+            uint64(700),
+            wallet_1_puzhash,
+            uint64(0),
+            puzzle_decorator_override=[{"decorator": "CLAWBACK", "clawback_timelock": 5}],
+        )
+        clawback_coin_id_2 = tx2.additions[0].name()
+        assert tx2.spend_bundle is not None
+        await wallet_1.push_transaction(tx2)
+        await full_node_api.wait_transaction_records_entered_mempool(records=[tx2])
+        await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(bytes32(b"\00" * 32)))
+        # Check merkle coins
+        await time_out_assert(
+            20, wallet_node_1.wallet_state_manager.coin_store.count_small_unspent, 2, 1000, CoinType.CLAWBACK
+        )
+        await time_out_assert(
+            20, wallet_node_2.wallet_state_manager.coin_store.count_small_unspent, 1, 1000, CoinType.CLAWBACK
+        )
+        assert await wallet_1.get_confirmed_balance() == 1999999998800
+        assert await wallet_2.get_confirmed_balance() == 0
         await asyncio.sleep(10)
         # clawback merkle coin
-        resp = await api_0.spend_clawback_coins(
-            dict({"coin_ids": [normal_puzhash.hex(), clawback_coin_id.hex()], "fee": 0})
-        )
+        resp = await api_1.spend_clawback_coins(dict({"coin_ids": [clawback_coin_id_1.hex()], "fee": 0}))
         json.dumps(resp)
         assert resp["success"]
         assert len(resp["transaction_ids"]) == 1
-        expected_confirmed_balance += await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
+        resp = await api_1.spend_clawback_coins(dict({"coin_ids": [clawback_coin_id_2.hex()], "fee": 0}))
+        assert resp["success"]
+        assert len(resp["transaction_ids"]) == 1
+        expected_confirmed_balance += await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet_1)
         await time_out_assert(
-            20, wallet_node.wallet_state_manager.coin_store.count_small_unspent, 0, 1000, CoinType.CLAWBACK
+            20, wallet_node_1.wallet_state_manager.coin_store.count_small_unspent, 0, 1000, CoinType.CLAWBACK
+        )
+        await time_out_assert(
+            20, wallet_node_2.wallet_state_manager.coin_store.count_small_unspent, 0, 1000, CoinType.CLAWBACK
         )
+        assert wallet_node_1.wallet_state_manager is not None
+        assert wallet_node_2.wallet_state_manager is not None
```

Comment:
It is for mypy check

Reply:
Like i said, it will never be `None` so this "mypy check" makes no sense and you can just drop those lines. 

https://github.com/Chia-Network/chia-blockchain/blob/226c6017649a0efc713bb6f390cddb8c085f530f/chia/wallet/wallet_node.py#L167-L173

Reply:
Resolving per discussion with @xdustinface and @ytx1991